lib/puppet/reports/datadog_reports.rb

```diff
@@ -127,5 +127,9 @@ def process
                                       :priority => event_priority,
                                       :source_type_name => 'puppet'
                                       ), :host => @msg_host)
+
+    Puppet.debug "Sending service check #{@msg_host} to Datadog"
+    service_check_status = if @status == 'failed' then 2 else 0 end
+    @dog.service_check('config_management.run.ok', @msg_host, service_check_status, :message => event_title)
```

Comment:
Yeah I agree, I chose this to keep it consistent with the existing event type `config_management.run`. It would be better to make it more explicit though.

How about `puppet.run.ok`?

Reply:
sounds good to me! Thanks @djova! 

We should make this configurable defaulting to disabled, I'm not sure I wan't want to enable this behavior by default just yet (we might do it in subsequent versions) what do you think?

Reply:
> We should make this configurable defaulting to disabled (we can read it from configfile), I'm not sure I wan't want to enable this behavior by default just yet (we might do it in subsequent versions) what do you think?

Yeah that's a good point. It turns out that we don't need this anymore. After testing it for a while we decided that this server-side reporting done by the puppet server isn't good enough for us so we've implemented some client-side reporting on every host where the puppet agent runs. 

Therefore we can close this now as we don't be using it.